### ISSUE TITLE
Consider using the native font stack that WP admin uses

### ISSUE THREAD

**Issue by demetris** [johnbillion/query-monitor#343]
Congratulations on the new major release and on the continuous impovement of the project!

Would you consider a pull request to replace the current font stack in the new version with the native font stack that is used by the WordPress admin?

Font stack used by Query Monitor:

``'Helvetica Neue', 'Lucida Grande', 'Arial', sans-serif``

Font stack used by the WordPress admin:

``-apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif``

Thank you for your time!

**Comment by johnbillion** [comment]
What a coincidence! I started playing around with this last night.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/208434/40420304-f0ca4884-5e7f-11e8-9ef0-1ba3ea8ae0e9.png">

**Comment by demetris** [comment]
I don’t know. I don’t see anything major.

On Windows, where the switch to the native font stack will have, I think, the most pronounced effect (from Arial to Segoe UI), when I first changed the font family the fonts appeared a bit small to me. But I got used to them almost immediately.

See my before and after screenshots from Query Monitor 3.0.1 on Windows 8.1 and Firefox 62, first with the current font stack, and then with the native font stack:

![20180523-141624-query-monitor-301-windows-81-firefox-62-default-font-stack](https://user-images.githubusercontent.com/31715/40422471-1e0623fa-5e98-11e8-804d-63969d6785df.png)

![20180523-141719-query-monitor-301-windows-81-firefox-62-native-font-stack](https://user-images.githubusercontent.com/31715/40422483-285c9d70-5e98-11e8-97cd-0fe180125d3c.png)

(The Query Monitor CSS reset sets font-size to 12px and line-height to 18px, while the WP admin has  13px and 18px.)

If it was up to me, I would not change anything else now.
